pkg/kubelet/container/sync_result.go

```diff
@@ -45,6 +45,8 @@ var (
 	ErrConfigPodSandbox = errors.New("ConfigPodSandboxError")
 	// ErrKillPodSandbox returned when runtime failed to stop pod's sandbox.
 	ErrKillPodSandbox = errors.New("KillPodSandboxError")
+	// ErrResizePod returned when runtime failed to resize a pod.
+	ErrResizePod = errors.New("ResizePodError")
```

Comment:
I don't think you need this (see comments where it's used)

Reply:
https://github.com/kubernetes/kubernetes/pull/130902#discussion_r2004186943